wsdl/rfd/RFDFormArchiver.wsdl

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<wsdl:definitions
+    name="RFDFormArchiver"
+    targetNamespace="urn:ihe:iti:rfd:2007"
+    xmlns="http://schemas.xmlsoap.org/wsdl/"

```

Comment:
RFD precisely specifies exact XML namespace prefixes (not just URIs), despite the fact that they are effectively irrelevant; whatever, complying :)